test/blackbox-tests/test-cases/describe/describe.t

```diff
@@ -1316,33 +1316,34 @@ Test other formats
 Test errors
 -----------
 
-  $ dune describe --lang 0.1 workspac
-  Error: Unknown constructor workspac
-  Hint: did you mean workspace?
+  $ dune describe workspacw --lang 0.1 
+  dune: unknown command 'workspacw', did you mean 'workspace'?
+  Usage: dune describe [COMMAND] …
+  Try 'dune describe --help' or 'dune --help' for more information.
   [1]
 
-  $ dune describe --lang 0.1 workspace xxx
+  $ dune describe workspace --lang 0.1 xxx
   Error: No such file or directory: xxx
   [1]
 
   $ touch yyy
-  $ dune describe --lang 0.1 workspace yyy
+  $ dune describe workspace --lang 0.1 yyy
   Error: File exists, but is not a directory: yyy
   [1]
 
-  $ dune describe --lang 1.0
+  $ dune describe workspace --lang 1.0
   dune: Only --lang 0.1 is available at the moment as this command is not yet
         stabilised. If you would like to release a software that relies on the output
         of 'dune describe', please open a ticket on
         https://github.com/ocaml/dune.
-  Usage: dune describe [OPTION]… [STRING]…
-  Try 'dune describe --help' or 'dune --help' for more information.
+  Usage: dune describe workspace [OPTION]… [DIRS]…
+  Try 'dune describe workspace --help' or 'dune --help' for more information.
   [1]
 
 opam file listing
 -----------------
 
-  $ dune describe --lang 0.1 opam-files | dune_cmd expand_lines
+  $ dune describe opam-files | dune_cmd expand_lines
```

Comment:
Why was the argument removed?

Reply:
We should probably just accept it everywhere we used to accept it. Otherwise it's a breaking change and I'd rather not think about break anyone's workflow for now.

Reply:
I can add a dummy option back in to `dune describe opam-files` but it cannot be in the same position. Unfortunately, command groups do not take options.

Reply:
Can't it be added individually to every subcommand?

Reply:
Yes, I have done that for `dune describe opam-files`. I meant that it isn't possible to keep the same position in the tests:
```
dune describe --lang 0.1 opam-files
```
Do you want me to add the lang arg to the `dune describe pp` and `dune describe external-lib-deps` too?

Reply:
I think that would be the safest probably. So let's do that.